Not finding AudioUnitSDK files inside of juce_audio_plugin_client module

I converted a Projucer plugin project to CMake using FRUT; the VST3 builds just fine but the AudioUnit fails to build with this error:
```
In file included from /Users/hayden/Documents/dev/Romplur-Main/RomplurPlugin/build/JuceLibraryCode/include_juce_audio_plugin_client_AU_1.mm:9:
In file included from /Users/hayden/Documents/dev/Romplur-Main/JUCE/modules/juce_audio_plugin_client/juce_audio_plugin_client_AU_1.mm:26:
In file included from /Users/hayden/Documents/dev/Romplur-Main/JUCE/modules/juce_audio_plugin_client/AU/juce_AU_Wrapper.mm:55:
/Users/hayden/Documents/dev/Romplur-Main/JUCE/modules/juce_audio_plugin_client/AU/AudioUnitSDK/MusicDeviceBase.h:8:10: fatal error: 'AudioUnitSDK/AUMIDIBase.h' file not found
#include <AudioUnitSDK/AUMIDIBase.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[3]: *** [CMakeFiles/RomplurPlugin_AU.dir/JuceLibraryCode/include_juce_audio_plugin_client_AU_1.mm.o] Error 1
```
All the files are there within the JUCE installation, and building the AU from the Projucer still works. The audio plugin client JUCE module is getting included in the FRUT-generated CMakeLists.txt like so:
```
jucer_project_module(
  juce_audio_plugin_client
  PATH "${JUCE_MODULES_GLOBAL_PATH}"
  JUCE_VST3_CAN_REPLACE_VST2 OFF
  # JUCE_FORCE_USE_LEGACY_PARAM_IDS
  # JUCE_FORCE_LEGACY_PARAMETER_AUTOMATION_TYPE
  # JUCE_USE_STUDIO_ONE_COMPATIBLE_PARAMETERS
  # JUCE_AU_WRAPPERS_SAVE_PROGRAM_STATES
  # JUCE_STANDALONE_FILTER_WINDOW_USE_KIOSK_MODE
)
```
Is there some other place I need to make sure that the AudioUnitSDK headers get included? For context I'm on macOS Monterey using JUCE 7.0.4 and Unix Makefiles as a generator. Any help or insight appreciated.

--- a/README.rst
+++ b/README.rst
@@ -306,6 +306,7 @@ contributors:
       <a href="https://github.com/dimitrisudell"><img src="https://github.com/dimitrisudell.png" width="100"><br />Dimitri Sudell</a>
       <br />
       <a href="https://github.com/McMartin/FRUT/issues?q=is%3Aissue+author%3Adimitrisudell" title="Bug reports">🐛</a>
+      <a href="https://github.com/McMartin/FRUT/pulls?q=state%3Amerged+reviewed-by%3Adimitrisudell+-author%3Adimitrisudell" title="Pull Request reviews">👀</a>
     </td>
     <td>
       <a href="https://github.com/danra"><img src="https://github.com/danra.png" width="100"><br />Dan Raviv</a>

--- a/README.rst
+++ b/README.rst
@@ -380,6 +380,23 @@ contributors:
       <a href="https://github.com/McMartin/FRUT/pulls?q=state%3Amerged+author%3ARiuzakiii" title="Code">💻</a>
     </td>
   </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/hsetlik"><img src="https://github.com/hsetlik.png" width="100"><br />Hayden Setlik</a>
+      <br />
+      <a href="https://github.com/McMartin/FRUT/issues?q=is%3Aissue+author%3Ahsetlik" title="Bug reports">🐛</a>
+    </td>
+    <td>
+    </td>
+    <td>
+    </td>
+    <td>
+    </td>
+    <td>
+    </td>
+    <td>
+    </td>
+  </tr>
   </tbody>
   </table>
 

--- a/cmake/Reprojucer.cmake
+++ b/cmake/Reprojucer.cmake
@@ -1,4 +1,4 @@
-# Copyright (C) 2016-2022  Alain Martin
+# Copyright (C) 2016-2023  Alain Martin
 # Copyright (C) 2017 Matthieu Talbot
 # Copyright (C) 2018-2019 Scott Wheeler
 # Copyright (C) 2022  Thiébaud Fuchs
@@ -2769,6 +2769,14 @@ function(jucer_project_end)
       endforeach()
       if(DEFINED rez_inputs)
         _FRUT_add_Rez_command_to_AU_plugin(${au_target} ${rez_inputs})
+      endif()
+      if(NOT (DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 7.0.3))
+        set(juce_audio_plugin_client_dir
+          "${JUCER_PROJECT_MODULE_juce_audio_plugin_client_PATH}/juce_audio_plugin_client"
+        )
+        target_include_directories(${au_target} PRIVATE
+          "${juce_audio_plugin_client_dir}/AU"
+        )
       endif()
 
       _FRUT_generate_plist_file(${au_target} "AU" "BNDL" "????")

